package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "postinstall": "bower install",
+    "oldpostinstall": "bower install",
```

Comment:
If this isn't necessary anymore, we might as well remove it.

Reply:
I thought we still needed the bower stuff for some samples, but on checking, I found it was no longer used. Yay!.